documentation/jetty-documentation/src/main/asciidoc/contribution-guide/4-build.adoc

```diff
@@ -24,11 +24,11 @@ Here are the minimum Maven and JDK version build requirements for each actively
 
 .Build versioning requirements by branch
 |===
-| Branch | Maven Version | Minimum JDK
+| Branch | Maven Version | Minimum JDK | Recommended JDK
 
-| jetty-12.0.x | Maven 3.8.6+  | OpenJDK 17+
-| jetty-11.0.x | Maven 3.8.6+  | OpenJDK 11+
-| jetty-10.0.x | Maven 3.8.6+  | OpenJDK 11+
+| jetty-12.0.x | Maven 3.9.2+  | OpenJDK 17+ | OpenJDK 19 (for optional quic using foreign memory)
+| jetty-11.0.x | Maven 3.8.6+  | OpenJDK 11+ | OpenJDK 17 (for optional virtual threads and HTTP/3 support)
+| jetty-10.0.x | Maven 3.8.6+  | OpenJDK 11+ | OpenJDK 17 (for optional virtual threads and HTTP/3 support)
```

Comment:
HTTP/3 works fine with JDK 11, 17 is only needed if you want to use the Foreign API binding instead of the JNA one.

Reply:
ah yes I can change that

Reply:
removed